copy.js

```diff
@@ -8,35 +8,53 @@
 
 var global_link_text = '';
 
-function maybe_clean (orig_url, regexp, f) {
+function glob_to_regexp (pattern) {
```

Comment:
wtf. javascript is truly an utter piece of shit.

Reply:
if it's of any consolation, also the RequestControl add-on uses [the same approach](https://github.com/tumpio/requestcontrol/blob/4e10ee9d16c8debe654596d5649c5d1bbac91a6c/src/RequestControl/api.js#L101) :-)